src/components/WebexAvatar/README.md

```diff
@@ -1,9 +1,22 @@
 # Webex Avatar Component
+Webex avatar component displays the avatar and status of a _Webex Teams_ user.
 
-Webex avatar component displays the avatar of a _Webex Teams_ user with a given `personID` and `adapter` object.
+![Webex Avatar](./WebexAvatar.png)
 
-## How to use
-
-```js
-<WebexAvatar personID="PersonID" adapter={peopleJSONAdapter} />
+## Preview
+To see all the different possible states of the Webex Avatar component, you can run our storybook
+```shell
+  npm start
 ```
+
+## Embed
+1. Create a component adapter from which the data will be retrieved (See [adapters](../../adapters)). For instance:
+    ```js
+      new jsonAdapter = new PeopleJSONAdapter()
+    ```
+2. Create a component instance by passing the person ID and the component data adapter
```

Comment:
any possibility we can specify the type of the parameters?

Reply:
Added parameter types as part of the instructions